src/utils/components/AddBootableVolumeModal/components/VolumeSource/hooks/useProjectsAndPVCs.ts

```diff
@@ -0,0 +1,25 @@
+import { IoK8sApiCoreV1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import useProjects from '@kubevirt-utils/hooks/useProjects';
+import usePVCs from '@kubevirt-utils/hooks/usePVCs';
+
+type useProjectsAndPVCsReturnType = {
+  error: Error;
+  projectsLoaded: boolean;
+  projectsNames: string[];
+  pvcs: IoK8sApiCoreV1PersistentVolumeClaim[];
+  pvcsLoaded: boolean;
+};
+
+export const useProjectsAndPVCs = (projectSelected: string): useProjectsAndPVCsReturnType => {
+  const [projectsNames, projectsLoaded, projectsErrors] = useProjects();
+
+  const [pvcs, pvcsLoaded, pvcsErrors] = usePVCs(projectSelected);
+
+  return {
+    error: projectsErrors || pvcsErrors,
+    projectsLoaded,
+    projectsNames,
+    pvcs,
+    pvcsLoaded,
+  };
+};
```

Comment:
isnt it redundent? if u already have 2 custom hoks, 1 for projects and other for pvcs, an u are combining loading, only errors.. i think this hook isn't a real must